Makefile

```diff
@@ -14,4 +14,9 @@ format:
 	cargo fmt -- --write-mode overwrite
 
 clean:
-	cargo clean
\ No newline at end of file
+	cargo clean
+
+examples: test
```

Comment:
This looks odd. Can we specify the target explicitly? Like `cargo build --example xxx`.